code/modules/power/singularity/singularity.dm

```diff
@@ -272,7 +272,7 @@
 	src.energy += gain
 	if(istype(A, /obj/machinery/power/supermatter_shard) && !consumedSupermatter)
 		desc = "[initial(desc)] It glows fiercely with inner fire."
-		name = "supermatter-charged [initial(name)]"
+		name = "Hawking-level [initial(name)]"

```

Comment:
Amusingly, the Hawking-level singularity is the one that gives off the least amount of Hawking radiation :)